src/components/Form/Form.Styled.ts

```diff
@@ -33,20 +34,39 @@ export const StyledDiv = styled.div`
 	align-items: flex-start;
 	width: 315px;
 	margin-bottom: 4em;
-	border: 2px solid #cecece;
-	border-radius: 10px;
-	overflow: hidden;
+`;
 
-	&:focus {
-		border: 3px solid white;
-	}
+export const StyledLabel = styled.label`
```

Comment:
Bardzo fajna semantyka, i te fieldsety, i legendy (tu bym sie zastanowił czy nie moglby to byc jakis h1/h2, ale chyba legend tez spoko, trzeba by byo poczytac specyfikacje)